bundle/edu.gemini.itc/src/main/java/edu/gemini/itc/gmos/IFUComponent.java

```diff
@@ -69,9 +69,9 @@ public IFUComponent(final String prefix, final double radius, final Boolean isIf
 
         int Nelements = 0;
         for (int i = 0; i < numX; i++) {
-            double x = (i - (numX/2.)) * (0.75*IFU_DIAMETER + IFU_SPACING);
+            double x = (i - (numX/2.)) * ((Math.sqrt(3)/2.)*IFU_DIAMETER + IFU_SPACING);
             for (int j = 0; j < numY; j++) {
-                double y = (2*j - (numY - Math.abs(i)%2 + 1)) * ((Math.sqrt(3)/4.)*IFU_DIAMETER + IFU_SPACING);
+                double y = (2*j - (numY - Math.abs(i)%2 + 1)) * (IFU_DIAMETER/2. + IFU_SPACING);
```

Comment:
A nitpicky point, but we've been trying to purge the code of things like `2.` for `double`s when we find them, and instead change them to `2.0` for clarity.

Otherwise, this looks fine to me. The only thing I can think of is that calculating `Math.sqrt(3)/2.0` in a loop is a bit redundant, as the value doesn't change, so it would be better to calculate it prior to the loop. Same with `IFU_DIAMETER/2.0`.

These are not serious issues, though.